src/zone.rs

```diff
@@ -105,8 +105,8 @@ fn get_international_names(tags: &Tags, default_name: &str) -> BTreeMap<String,
         .collect()
 }
 
-impl Zone {
-    pub fn default() -> Self {
+impl Default for Zone {
+    fn default() -> Self {
```

Comment:
can't it be derived?

Reply:
nop because of osmpbfreader::Tags :stuck_out_tongue_winking_eye: 

I'll make a PR later for this, but this will be for another PR

Reply:
That's because you use an old version of osmpbfreader :innocent:

Reply:
oups though I checked :confounded: 

but It needs an update of osm_boundary_utils too, so I'll be for later